src/api/CoinBase.ts

```diff
@@ -0,0 +1,63 @@
+import { ExchangeApi } from '../types/ExchangeApi';
+import fetch, { Response } from 'node-fetch';
+import BigNumber from 'bignumber.js';
+
+const URL = 'https://api.coinbase.com/v2/exchange-rates';
+
+export type CoinBaseResponse = {
+    data: {
+        currency: string;
+        rates: Record<string, string>;
+    };
+};
+
+const CURRENCY_MAPPING: Record<string, string> = {
+    RIF: 'RIF',
+    RBTC: 'RBTC',
+    TKN: 'RIF'
```

Comment:
I'd add a comment here to specify that this is for testing purposes only.
Or eventually, we could implement an ExchangeApi for the TestToken only in another task. WDYT?

Reply:
I think that the comment will be enough for now. 

The TestToken Exchange Api was my first approach but by implementing it, we would also need to change the way of how is implemented now since the ExchangeApi to be used, would be based on the token and therefore a selection process would be needed before.

Reply:
I will be updating with a different implementation, that could simplify it.

Reply:
updated